nevergrad/optimization/optimizerlib.py

```diff
@@ -396,6 +402,7 @@ def __init__(
 
 
 CMA = ParametrizedCMA().set_name("CMA", register=True)
+ECMA = ParametrizedCMA(elitist=True).set_name("ECMA", register=True)
```

Comment:
is it needed here? I would tend to move ECMA (and FCMA by the way) to experimentalvariants

Reply:
As you prefer; done.

Reply:
I keep FCMA actually due to a weird dependency in mix-simulator.